index.html

```diff
@@ -17,21 +17,26 @@
             <li class="navbar-toggle">
               <img src="img/Union.png" alt="menubar" />
             </li>
+            <li>Portoflio</li>
+            <li>About</li>
+            <li>Contact</li>
           </nav>
         </ul>
       </header>
       <section class="container">
-        <main>
+        <section class="maincontainer">
+        <main class="maintxt">
           <h1 class="headerone font">I’m Emmanuel Glad to see you!</h1>
           <p class="text font">
             I’m a software developer! I can help you build a product , feature
             or website Look through some of my work and experience! If you like
             what you see and have a project you need coded, don’t hestiate to
             contact me.
           </p>
-        </main>
+        
         <footer>
           <p class="connect font">LET’S CONNECT</p>
+        </main>
           <ul class="social">
             <li><img src="img/Twitter.png" alt="" /></li>
```

Comment:
- [ ] Kindly put the social handle in a tag so that when you click it will take that social page.

Reply:
Done.